python/flow_sdk/shim_airbyte_cdk.py

```diff
@@ -257,7 +257,11 @@ def open(self, open: request.Open, emit: t.Callable[[Response], None]) -> None:
             logger, config, catalog, state.to_airbyte_input()
         ):
             if record := message.record:
-                entry = index[(record.namespace, record.stream)]
+                entry = index.get((record.namespace, record.stream), None)
+                if entry is None:
+                    logger.warn(f"Document read in unrecognized stream {record.stream} (namespace: {record.namespace})")
```

Comment:
i assume this is temporary per your update? it has the potential to produce a lot of logs.

Reply:
Yeah, it's temporary. I just really didn't want to introduce a change that lets us accidentally read from disabled streams and then drop those records on the floor without complaining about it.